src/components/Modal/types.ts

```diff
@@ -87,6 +88,8 @@ type BaseModalProps = Partial<ModalProps> & {
 
     /** Should we apply padding style in modal itself. If this value is false, we will handle it in ScreenWrapper */
     shouldUseModalPaddingStyle?: boolean;
+
+    initialFocus?: FocusTargetOrFalse | undefined | (() => void);
```

Comment:
same as above. Can we use `HTMLElement | false` instead?

Reply:
That makes sense. We can use all the supporting types in that case. But, I still don't find a valid reason to use `FocusTargetOrFalse`